CMakeLists.txt

```diff
@@ -1,6 +1,6 @@
 cmake_minimum_required(VERSION 3.21)
-project(JIT_TEST LANGUAGES CXX C Fortran)
-set (CMAKE_CXX_STANDARD 17)
+project(JIT_TEST LANGUAGES CXX C Fortran CUDA)
+set (CMAKE_CXX_STANDARD 11)
```

Comment:
So C++17 does not work for CUDA?

Reply:
yeah, seems not to - both c++17 and c++14